packages/server/src/ChunkExtractor.js

```diff
@@ -187,8 +188,13 @@ class ChunkExtractor {
     this.outputPath = outputPath || this.stats.outputPath
     this.statsFile = statsFile
     this.entrypoints = Array.isArray(entrypoints) ? entrypoints : [entrypoints]
+    this.filter = filter || Boolean
```

Comment:
Let's use a less generic name on `this` scope. For example `filterChunk`

Reply:
You're right, I made the change.

Reply:
I also rename the option as it makes its purpose more explicit.